src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-notifications/notification-channels-control-where-send-alerts.mdx

```diff
@@ -64,11 +64,14 @@ These are the available notification channel types.
     id="user"
     title="User"
   >
-    For your convenience, we automatically load all users and their email addresses for the selected account. If your account has one or more [sub-accounts](/docs/accounts/install-new-relic/account-setup/manage-apps-or-users-sub-accounts), the notification channel includes only users for the [currently selected master or sub-account](/docs/accounts/install-new-relic/account-setup/use-multiple-accounts).
+    Requirements: 
+      * This feature is not available for users on the [New Relic One user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model/#user-models), only to users on the original user model. As a workaround, you can use the [email notification channel](#email). 
```

Comment:
It feels a little strange to me that you have a link on New Relic One user model and not on original user model. Maybe it's possible to make it a little clearer that that link will take you to a doc that discusses the difference between the two models?